hedera-node/configuration/testnet/upgrade/feeSchedules.json

```diff
@@ -3663,6 +3864,207 @@
+                        "sbh": 1,
+                        "gas": 81,
+                        "bpr": 12109,
+                        "sbpr": 303,
+                        "min": 0,
+                        "max": 1000000000000000
+                    },
+                    "networkdata": {
+                        "constant": 151489557,
+                        "bpt": 242186,
+                        "vpt": 605466012,
+                        "rbh": 161,
+                        "sbh": 12,
+                        "gas": 1615,
+                        "bpr": 242186,
+                        "sbpr": 6055,
+                        "min": 0,
+                        "max": 1000000000000000
+                    },
+                    "servicedata": {
+                        "constant": 151489557,
+                        "bpt": 242186,
+                        "vpt": 605466012,
+                        "rbh": 161,
+                        "sbh": 12,
+                        "gas": 1615,
+                        "bpr": 242186,
+                        "sbpr": 6055,
+                        "min": 0,
+                        "max": 1000000000000000
+                    }
+                },
+                {
+                    "subType": "TOKEN_FUNGIBLE_COMMON",
+                    "nodedata": {
+                        "constant": 7983519,
+                        "bpt": 12763,
+                        "vpt": 31908136,
+                        "rbh": 9,
+                        "sbh": 1,
+                        "gas": 85,
+                        "bpr": 12763,
+                        "sbpr": 319,
+                        "min": 0,
+                        "max": 1000000000000000
+                    },
+                    "networkdata": {
+                        "constant": 159670382,
+                        "bpt": 255265,
+                        "vpt": 638162730,
+                        "rbh": 170,
+                        "sbh": 13,
+                        "gas": 1702,
+                        "bpr": 255265,
+                        "sbpr": 6382,
+                        "min": 0,
+                        "max": 1000000000000000
+                    },
+                    "servicedata": {
+                        "constant": 159670382,
+                        "bpt": 255265,
+                        "vpt": 638162730,
+                        "rbh": 170,
+                        "sbh": 13,
+                        "gas": 1702,
+                        "bpr": 255265,
+                        "sbpr": 6382,
+                        "min": 0,
+                        "max": 1000000000000000
+                    }
+                },
+                {
+                    "subType": "TOKEN_FUNGIBLE_COMMON_WITH_CUSTOM_FEES",
+                    "nodedata": {
+                        "constant": 15967037,
+                        "bpt": 25527,
+                        "vpt": 63816270,
+                        "rbh": 17,
```

Comment:
We should add the canonical price to `canonical-prices.json`

Reply:
Added 👍
I'm guessing I should add the canonical price for the claim and cancel airdrop, right?

Reply:
yeah for all three of them @vtronkov